internal/bminventory/inventory.go

```diff
@@ -4521,38 +4520,6 @@ func (b *bareMetalInventory) GetPreflightRequirements(ctx context.Context, param
 	return installer.NewGetPreflightRequirementsOK().WithPayload(requirements)
 }
 
-func (b *bareMetalInventory) GetHostRequirements(_ context.Context, params installer.GetHostRequirementsParams) middleware.Responder {
-	requirements, err := b.hwValidator.GetDefaultVersionRequirements()
```

Comment:
Could `GetDefaultVersionRequirements` be removed as well? I don't see it being used.

Reply:
Sure

Reply:
Removed